rep-0008.rst

```diff
@@ -104,13 +168,15 @@ src/packagename helps encourage this.
 Python Features and Version
 ===========================
 
-The current target platform is Python 2.5 [2]_, though we wish to make
-code that is easily transitioned to Python 2.6, 2.7, and, more
-importantly, Python 3000 [3]_.
+The target version of Python changes based on the requirements for each distribution.
+See REP-0003 [2]_ for the target Python version for each ROS distribution.
+
+Currently, as of 2015, the target is Python 2.7+, but we wish to encourage
+code that is easily transitioned to Python 3 [3]_.
```

Comment:
We could add a reference to http://wiki.ros.org/python_2_and_3_compatible_code here.